bears/css/PostCSSBear.py

```diff
@@ -0,0 +1,29 @@
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+from coalib.settings.Setting import typed_list
+
+
+class PostCSSBear(Lint, LocalBear):
+    executable = "postcss"
+    diff_message = "PostCSS plugins in action."
+    gives_corrected = True
+
+    def run(self, filename, file, postcss_plugins: typed_list(str)=[]):
+        '''
+        This bear adds ``PostCSS`` utilities to analyze CSS code.
+
+        :param postcss_plugins:     List of PostCSS plugins to use with
+                                    this bear which can be defined in
+                                    .coafile.
+        '''
+        if postcss_plugins:
```

Comment:
Would be nicer to do `if not postcss_plugin` and then you don't need an else at all.
makes it easier to read